ProftaakPeriode4/ProftaakPeriode4/Mesh.h

```diff
@@ -153,7 +153,8 @@ class Mesh
 	* \param position The position to be drawn on
 	* \param rotation The rotation that it should be drawn on
 	* \param rotationAngle the rotation angle that it should be drawn on
+	* \param scale The scale it should be drawn on
 	*/
-	void Draw(Vec3f position, Vec3f rotation, float rotationAngle);
+	void Draw(Vec3f position, Vec3f rotation, float rotationAngle, Vec3f scale);
```

Comment:
I think it would be wise to make scale an optional parameter by initialising it as {1.0f, 1.0f, 1.0f}, same could be said for every parameter of the Draw method. Also, rotationAngle is redundant...